### PR TITLE
showning kbfs notifications

### PR DESCRIPTION
**Pull request by chrisnojima**
@keybase/react-hackers

### CODE REVIEW

File: desktop/renderer/notifier.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <script>
 const ipc = require('electron').ipcRenderer;
 
-ipc.on('notify', (title, opts) => {
-  new Notification(title, opts);
+ipc.on('notify', (event, title, opts) => {
+    new Notification(title, opts);
```
Comment (MarcoPolo) [inline desktop/renderer/notifier.html]: is the indent right?

File: react-native/react/native/notification-listeners.desktop.js [hunk 2]
```diff
@@ -1,7 +1,30 @@
 import notify from '../../../desktop/app/hidden-window-notifications'
+import enums from '../../react/keybase_v1'
 
 export default {
   'keybase.1.NotifySession.loggedOut': () => {
     notify('Logged Out')
+  },
+  'keybase.1.NotifyFS.FSActivity': ({notification}) => {
```
Comment (MarcoPolo) [inline react-native/react/native/notification-listeners.desktop.js]: :sheep: It would be really cool if notification was typed :)

**Comment by MarcoPolo** [comment]
:+1: % annotation request (optional)

**Comment by chrisnojima** [comment]
@keybase/react-hackers take one more peek

File: .editorconfig [hunk 3]
```diff
@@ -0,0 +1,24 @@
+root = true
```
Comment (chrisnojima) [inline .editorconfig]: Adding this so html spacing / etc gets picked up

File: .eslintrc [hunk 4]
```diff
@@ -1,6 +1,9 @@
 {
     "parser": "babel-eslint",
     "extends": ["standard", "standard-react"],
+    "plugins": [
```
Comment (chrisnojima) [inline .eslintrc]: This allows us to do flow stuff w/ the other configs we need (arrows etc)

File: protocol/Makefile [hunk 5]
```diff
@@ -79,6 +83,6 @@ clean:
 deps:
 	npm install
 
-build: build-stamp ../go/protocol/keybase_v1.go js/keybase_v1.js
+build: build-stamp ../go/protocol/keybase_v1.go js/keybase_v1.js js/flow-types.js
```
Comment (chrisnojima) [inline protocol/Makefile]: Call this script when we do a build

File: react-native/react/actions/login.js [hunk 6]
```diff
@@ -3,7 +3,7 @@ import {isMobile} from '../constants/platform'
 import QRCodeGen from 'qrcode-generator'
 import {navigateTo, routeAppend, getCurrentURI, getCurrentTab} from './router'
 import engine from '../engine'
-import enums from '../keybase_v1'
+import enums from '../constants/types/keybase_v1'
```
Comment (chrisnojima) [inline react-native/react/actions/login.js]: Making keybase_v1 like flow-types

**Comment by MarcoPolo** [comment]
:+1:

**Comment by cjb** [comment]
:thumbsup:

**Pull request merged**